src/command-instance.js

```diff
@@ -5,19 +5,18 @@ const SSH = require("./ssh");
 
 module.exports = class CommandInstance {
     constructor() {
+        this.ssh = new SSH();
+
         this.fileName = this.normalizePath(vscode.window.activeTextEditor.document.fileName);
```

Comment:
why do you have 2 subsequent this.fileName assignments?

Reply:
First assignment is generating the file name, second assignment is remapping the paths **when SSH testing is enabled**.